index.js

```diff
@@ -46,6 +48,8 @@ const toJSON = from => {
 	return json;
 };
 
+const getErrorConstructor = name => errorConstructors.get(name) || Error;
```

Comment:
I tried ?? but it fails in Node 12. If desired, Node 12 support can probably be dropped separately so it appears in the changelog